### PR TITLE
Look for 1 min and 1 max NB EPs with 4.5 and lower clusters

### LINKED ISSUE #3085: Deployment is failing that it expects 2 noobaa endpoints, but only one exists
Job:
https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/13233//consoleFull

```
12:59:42 - MainThread - ocs_ci.ocs.ocp - INFO - status of  at column STATUS - item(s) were ['Running'], but we were waiting for all 2 of them to be Running
12:59:42 - MainThread - ocs_ci.utility.utils - INFO - Going to sleep for 3 seconds before next iteration
12:59:45 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/jenkins/current-cluster-dir/openshift-cluster-dir/auth/kubeconfig get Pod  -n openshift-storage --selector=noobaa-s3=noobaa -o yaml
12:59:45 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/jenkins/current-cluster-dir/openshift-cluster-dir/auth/kubeconfig get Pod noobaa-endpoint-7646c966b4-q92mc -n openshift-storage
12:59:45 - MainThread - ocs_ci.ocs.ocp - INFO - status of  at column STATUS - item(s) were ['Running'], but we were waiting for all 2 of them to be Running
12:59:45 - MainThread - ocs_ci.ocs.ocp - ERROR - timeout expired: Timed Out: (600,)
12:59:45 - MainThread - ocs_ci.ocs.ocp - ERROR - Wait for Pod resource  at column STATUS to reach desired condition Running failed, last actual status was ['Running']
12:59:45 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/jenkins/current-cluster-dir/openshift-cluster-dir/auth/kubeconfig describe Pod  --selector=noobaa-s3=noobaa

....
tests/ecosystem/deployment/test_deployment.py:25: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
ocs_ci/ocs/resources/storage_cluster.py:185: in ocs_install_verification
    timeout=timeout
ocs_ci/ocs/ocp.py:595: in wait_for_resource
    raise(ex)
ocs_ci/ocs/ocp.py:504: in wait_for_resource
    timeout, sleep, self.get, resource_name, True, selector
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <ocs_ci.utility.utils.TimeoutSampler object at 0x7fad989f5668>

    def __iter__(self):
        if self.start_time is None:
            self.start_time = time.time()
        while True:
            self.last_sample_time = time.time()
            try:
                yield self.func(*self.func_args, **self.func_kwargs)
            except Exception as ex:
                msg = f"Exception raised during iteration: {ex}"
                logging.error(msg)
            if self.timeout < (time.time() - self.start_time):
>               raise self.timeout_exc_cls(*self.timeout_exc_args)
E               ocs_ci.ocs.exceptions.TimeoutExpiredError: Timed Out: (600,)

ocs_ci/utility/utils.py:796: TimeoutExpiredError
---------------------------- Captured stdout setup -----------------------------```

### PR DESCRIPTION
**Pull request by ebenahar**
Fixes #3085 

Signed-off-by: Elad Ben Aharon <ebenahar@redhat.com>

### CODE REVIEW

**Review by Avilir — APPROVED** [review]

File: ocs_ci/ocs/resources/storage_cluster.py [hunk 1]
```diff
@@ -139,13 +139,8 @@ def ocs_install_verification(
...
 
-    # Fetch the min and max Noobaa endpoints from the run config
-    if check_nodes_specs(min_cpu=constants.MIN_NODE_CPU, min_memory=constants.MIN_NODE_MEMORY):
-        min_eps = config.DEPLOYMENT.get('min_noobaa_endpoints')
-        max_eps = config.DEPLOYMENT.get('max_noobaa_endpoints')
-    else:
-        min_eps = 1
-        max_eps = 1 if float(config.ENV_DATA['ocs_version']) < 4.6 else 2
+    min_eps = 1
+    max_eps = 2 if float(config.ENV_DATA['ocs_version']) >= 4.6 else 1
```
Comment (petr-balogh) [inline ocs_ci/ocs/resources/storage_cluster.py:143]:
```suggestion
    max_eps = config.DEPLOYMENT.get('max_noobaa_endpoints') if float(config.ENV_DATA['ocs_version']) >= 4.6 else 1
```

But this is too long so maybe more readable will be 
```suggestion
    max_eps = config.DEPLOYMENT.get('max_noobaa_endpoints') 
    if float(config.ENV_DATA['ocs_version']) < 4.6:
        max_eps = 1
```

File: ocs_ci/ocs/resources/storage_cluster.py [hunk 2]
```diff
@@ -139,13 +139,8 @@ def ocs_install_verification(
...
         rgw_count = 1
 
-    # Fetch the min and max Noobaa endpoints from the run config
-    if check_nodes_specs(min_cpu=constants.MIN_NODE_CPU, min_memory=constants.MIN_NODE_MEMORY):
-        min_eps = config.DEPLOYMENT.get('min_noobaa_endpoints')
-        max_eps = config.DEPLOYMENT.get('max_noobaa_endpoints')
-    else:
-        min_eps = 1
-        max_eps = 1 if float(config.ENV_DATA['ocs_version']) < 4.6 else 2
+    min_eps = 1
```
Comment (petr-balogh) [inline ocs_ci/ocs/resources/storage_cluster.py:142]: Currently you don't do check for min_eps also for OCS < 4.6, is that intention as well?

File: ocs_ci/ocs/resources/storage_cluster.py (hunk 1 above)
Comment (ebenahar) [reply ocs_ci/ocs/resources/storage_cluster.py:143]: we should not use config's max_noobaa_endpoints anymore

we don't want to change the CR manually anymore

unless there will be a specific test for it, but this test will change it and not from config

we can remove these keys from default_config

File: ocs_ci/ocs/resources/storage_cluster.py (hunk 2 above)
Comment (ebenahar) [reply ocs_ci/ocs/resources/storage_cluster.py:142]: it should be 1 in all versions

**Comment by petr-balogh** [comment]
Two production tier1 jobs triggered for 4.5.1 build:
https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-trigger-aws-ipi-3az-rhcos-3m-3w-tier1/92/console
https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-trigger-vsphere-upi-1az-rhcos-vsan-3m-3w-tier1/61/

**Review by petr-balogh — APPROVED** [review]

**Review by dahorak — APPROVED** [review]

**Review by vavuthu — APPROVED** [review]

**Pull request merged**